### PR TITLE
chore(angular): update ngrx to 11.1.0 while migration

### LINKED ISSUE #5261: Migration to Nx > 11.0.0 should update ngrx to 11.1.0
There is an issue with NgRx 11 https://github.com/ngrx/platform/issues/2919  which is fixed in 11.0.1 as per the comments.

If you can help me where to make changes , I will be more than happy to raise a PR.

## Current Behavior
Migration to Nx does not update ngrx

## Expected Behavior
It should update ngrx version as well.

### PR DESCRIPTION
**Pull request by santoshyadavdev**
Fixes #5261

## Related Issue(s)

Fixes #

### CODE REVIEW

**Comment by santoshyadavdev** [comment]
Do i need to update some test cases?

File: packages/angular/migrations.json [hunk 1]
```diff
@@ -411,6 +411,36 @@
...
+        "alwaysAddToPackageJson": false
+      },
+      "@ngrx/store-devtools": {
+        "version": "11.1.0",
+        "alwaysAddToPackageJson": false
+      },
+      "@ngrx/component-store": {
+        "version": "11.1.0",
+        "alwaysAddToPackageJson": false
+      },
```
Comment (leosvelperez) [inline packages/angular/migrations.json:443]: @santoshyadavdev the extra comma here at the end is what seems to be causing the tests to fail

Comment (santoshyadavdev) [reply packages/angular/migrations.json:443]: Ahh, thanks 😀 let me update.

**Comment by santoshyadavdev** [comment]
Thanks a lot @brandonroberts @leosvelperez for help, the CI looks green now.

**Review by brandonroberts — APPROVED** [review]

**Review by leosvelperez — APPROVED** [review]
LGTM! 🎉 
@santoshyadavdev could you please rebase and push to solve the conflicts?

**Comment by santoshyadavdev** [comment]
> LGTM! 🎉
> @santoshyadavdev could you please rebase and push to solve the conflicts?

Done @leosvelperez .

**Pull request merged**